apps/docs/src/app/core/component-docs/bar/bar-docs.component.html

```diff
@@ -104,3 +104,11 @@
     <fd-bar-page-responsive-example></fd-bar-page-responsive-example>
 </component-example>
 <code-example [exampleFiles]="barPageResponsiveExample"></code-example>
+
+<fd-docs-section-title id="custom-colors" componentName="bar"> Adding Custom Colors </fd-docs-section-title>
```

Comment:
Actually our goal is not to show how they can add custom colors, but how to build shellbar with completely custom content as opposed to the default implementation